spec/unit/settings/ini_file_spec.rb

```diff
@@ -4,6 +4,13 @@
 require 'puppet/settings/ini_file'
 
 describe Puppet::Settings::IniFile do
+  # different UTF-8 widths
+  # 1-byte A
+  # 2-byte ۿ - http://www.fileformat.info/info/unicode/char/06ff/index.htm - 0xDB 0xBF / 219 191
+  # 3-byte ᚠ - http://www.fileformat.info/info/unicode/char/16A0/index.htm - 0xE1 0x9A 0xA0 / 225 154 160
+  # 4-byte 𠜎 - http://www.fileformat.info/info/unicode/char/2070E/index.htm - 0xF0 0xA0 0x9C 0x8E / 240 160 156 142
+  let (:mixed_utf8) { "A\u06FF\u16A0\u{2070E}" } # Aۿᚠ𠜎
```

Comment:
I haven't seen `{`..`}` before in the unicode representation. `\u2070E` and `\u{2070E}` are indeed different characters - what do the brackets mean?